src/markdoc/partials/quickstarts/js-quickstart.md

```diff
@@ -95,22 +95,26 @@ Create a new Node.js project and install the Golem SDK by entering the following
 mkdir try_golem
 cd try_golem
 npm init
-npm install @golem-sdk/task-executor
+npm install @golem-sdk/golem-js
+npm install @golem-sdk/pino-logger
```

Comment:
I wonder if it would not be better to use our cli here, e.g. `npx golem-sdk new` .. ?

Reply:
we would need to change it in all examples